src/test/java/net/datafaker/service/FakeValuesServiceTest.java

```diff
@@ -35,8 +40,11 @@
 
 class FakeValuesServiceTest extends AbstractFakerTest {
 
-    private static final Long MILLIS_IN_AN_HOUR = 1000 * 60 * 60L;
-    private static final Long MILLIS_IN_A_DAY = MILLIS_IN_AN_HOUR * 24;
+    private static final Long SECONDS_IN_5_HOURS = TimeUnit.HOURS.toSeconds(5);
+    private static final Long SECONDS_IN_10_DAYs = TimeUnit.DAYS.toSeconds(10);
```

Comment:
was the lowercase `s` her intentional?

Reply:
don't think so, it is just a private const, so seems just a misprint